react/features/recording/components/LiveStream/GoogleSigninForm.native.js

```diff
@@ -0,0 +1,240 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
+import { connect } from 'react-redux';
+import {
+    GoogleSignin
+} from 'react-native-google-signin';
```

Comment:
Maybe google signing should be wrapped within google-api? It seems like they are parallel and may be conceptually sticky.

Reply:
Good idea, thanks!